test/vanilla/AutoRestRFC1123DateTimeTestService.java

```diff
@@ -1,37 +0,0 @@
-/**
```

Comment:
I have no idea how these files were around--they were never being compiled, but I just deleted them at some point and that deletion stuck around in here.